### PR TITLE
Resize cache

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
#756

10Mb is definitively too few for a cache when the user has many images, especially when #982 is merged.

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
@AndyScherzinger @LukasReschke please review :)

File: src/com/owncloud/android/ui/activity/Preferences.java [hunk 1]
```diff
@@ -237,6 +238,28 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
...
+            Long cacheSize = ThumbnailsCacheManager.getMaxSize();
+            pCacheSize.setSummary(cacheSize + " Mb");
+            pCacheSize.setOnPreferenceChangeListener(new OnPreferenceChangeListener() {
+                @Override
+                public boolean onPreferenceChange(Preference preference, Object newValue) {
+                    String temp = (String) newValue;
+                    Long size = Long.decode(temp);
+                    if (ThumbnailsCacheManager.setMaxSize(size)){
+                        appPrefs.edit().putString("pref_cache_size", size.toString());
+                        pCacheSize.setSummary(size + " Mb");
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/ui/activity/Preferences.java]: We use a capital B for Byte on the lists so for visual consistency I would use " MB" but I leave that to @jancborchardt

File: src/com/owncloud/android/ui/activity/Preferences.java [hunk 2]
```diff
@@ -237,6 +238,28 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
...
+        if (pCacheSize != null){
+            final SharedPreferences appPrefs =
+                    PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+            Long cacheSize = ThumbnailsCacheManager.getMaxSize();
+            pCacheSize.setSummary(cacheSize + " Mb");
+            pCacheSize.setOnPreferenceChangeListener(new OnPreferenceChangeListener() {
+                @Override
+                public boolean onPreferenceChange(Preference preference, Object newValue) {
+                    String temp = (String) newValue;
+                    Long size = Long.decode(temp);
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/ui/activity/Preferences.java]: The two lines can be combined since temp isn't used anywhere else.

**Comment by AndyScherzinger** [comment]
Looks good to me :+1:

File: src/com/owncloud/android/ui/activity/Preferences.java (hunk 1 above)
Comment (tobiasKaminsky) [inline src/com/owncloud/android/ui/activity/Preferences.java]: You are right!
Mb is also official Megabit, which is not the same as MB = MegaByte...

**Pull request closed without merging**